rdm/checklists/62304_base_class_a.txt

```diff
@@ -0,0 +1,103 @@
+# Common base Audit checklist for IEC62304 Class A products
+62304:4.1 Quality Management System
+62304:4.2 Risk Management
+62304:4.3.a Software safety classification: system class
+# 62304:4.3.b Software safety classification: risk control (dropped by AMD)
```

Comment:
Here is one more idea for the checklist formats: What if, when the first character on the line was a `!`, it removed the item if it exists earlier in the list?

Reply:
@yujanshrestha @orwonthe what do you guys think of this extension? Is it making things too complicated?

Reply:
It might have some future use. If the next version of some standard drops a requirement (which did happen with AMD1), there would be no need to alter the earlier checklists already in use. There would only be a need to add a new checklist that include the prior with some additions, subtractions, and replacements.